Dockerfile.ubuntu

```diff
@@ -181,12 +181,25 @@ RUN if [ "$ov_use_binary" = "1" ] && [ "$DLDT_PACKAGE_URL" = "" ] ; then true ;
 RUN if [ -f /opt/intel/openvino/samples/cpp/build_samples.sh ];  then /opt/intel/openvino/samples/cpp/build_samples.sh ; fi
 #################### END OF OPENVINO BINARY INSTALL
 
-# install opencv
-ARG OPENVINO_OPENCV_DOWNLOAD_SERVER=""
-RUN if [ "$ov_use_binary" = "1" ] && [ "$DLDT_PACKAGE_URL" != "" ]; then true ; else exit 0 ; fi ; \
-    cd /opt/intel/openvino/ && echo URL ${OPENVINO_OPENCV_DOWNLOAD_SERVER}/openvino_opencv_ubuntu20.tgz && \
-    curl ${OPENVINO_OPENCV_DOWNLOAD_SERVER}/openvino_opencv_ubuntu20.tgz -o ${TMPDIR:-/tmp}/openvino_opencv_linux.tgz && \
-    tar -xf ${TMPDIR:-/tmp}/openvino_opencv_linux.tgz
+# Build OpenCV from source (for binary inputs and custom nodes)
```

Comment:
it might be better to move it up after AWS S3 SDK to take advantage of docker cache

Reply:
moved